zeroex/ordervalidator/order_validator_bench_test.go

```diff
@@ -0,0 +1,74 @@
+	"github.com/0xProject/0x-mesh/zeroex"
+	"github.com/ethereum/go-ethereum/common"
+	ethrpc "github.com/ethereum/go-ethereum/rpc"
+)
+
+const (
+	defaultEthRPCTimeout = 5 * time.Second
+)
+
+var (
+	signedOrders   []*zeroex.SignedOrder
+	orderValidator *OrderValidator
+)
+
+func init() {
+	rpcClient, err := ethrpc.Dial(constants.GanacheEndpoint)
+	if err != nil {
+		panic(err)
+	}
+	ethRPCClient, err := ethrpcclient.New(rpcClient, defaultEthRPCTimeout, ratelimit.NewUnlimited())
+	if err != nil {
+		panic(err)
+	}
+	orderValidator, err = New(ethRPCClient, constants.TestChainID, constants.TestMaxContentLength, ethereum.GanacheAddresses)
+	if err != nil {
+		panic(err)
+	}
+
+	multiAssetAssetData := common.Hex2Bytes("94cfcdd7000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000030000000000000000000000000000000000000000000000000000000000000046000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000120000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000001400000000000000000000000000000000000000000000000000000000000000024f47261b00000000000000000000000001dc4c1cefef38a777b15aa20260a54e584b16c48000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000044025717920000000000000000000000001dc4c1cefef38a777b15aa20260a54e584b16c480000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000204a7cb5fb70000000000000000000000001dc4c1cefef38a777b15aa20260a54e584b16c480000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000001800000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000006400000000000000000000000000000000000000000000000000000000000003e90000000000000000000000000000000000000000000000000000000000002711000000000000000000000000000000000000000000000000000000000000000300000000000000000000000000000000000000000000000000000000000000c800000000000000000000000000000000000000000000000000000000000007d10000000000000000000000000000000000000000000000000000000000004e210000000000000000000000000000000000000000000000000000000000000044025717920000000000000000000000001dc4c1cefef38a777b15aa20260a54e584b16c4800000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000")
```

Comment:
Nit: `multiAssetAssetData` => `multiAssetData` (at least, this is what I would normally call this. I can understand a difference of opinions on this).

Reply:
I'd like to see some benchmarks on more normal asset data than this. This is a pretty pathological case that will be significantly harder on the ABI decoder than normal asset data. It would be nice to see what the time difference is on an order that only uses `erc20AssetData` for `makerAssetData` and `takerAssetData` and has empty fee asset data (this much closer to the average case for an order that is added to Mesh).

Reply:
There is no ABI decoding occurring, only encoding. Only the length of the data should affect the encoding time, not the contents.

Reply:
What I'm saying is that we should have more realistic benchmarks. These asset data lengths are not very realistic or the average case.

My mistake on the decoding vs encoding. I don't think that changes my suggestion though as this will be more taxing to encode as well.